scripts/game.js

```diff
@@ -0,0 +1,116 @@
+function startNewGame() {
+  if (players[0].name === "" || players[1].name === "") {
+    playerNameAlertModal.style.display = "block";
+    return;
+  }
+
+  resetGameStatus();
+
+  activeGameSection.style.display = "block";
+  playerTurnElement.textContent = players[activePlayer].name;
+}
+
+function switchPlayer() {
+  if (activePlayer === 0) {
```

Comment:
Have you learnt about ternary operators yet?

Reply:
No

Reply:
e.g.  `activePlayer = activePlayer === 0 ? 1 : 0`